addons/web/static/src/search/favorite_menu/favorite_menu.xml

```diff
@@ -19,7 +19,7 @@
                         >
                         <span class="d-flex p-0 align-items-center justify-content-between">
                             <t t-esc="item.description"/>
-                            <i class="o_icon_right fa fa-trash-o"
+                            <i class="pl-1 o_icon_right fa fa-trash-o"
```

Comment:
A `margin-left` instead of `padding-left` would make more sense in this case: the point is to add spacing *between* the text and the icon, as a `padding` would add it *inside* the icon.